### ISSUE TITLE
attempt to call method 'enableAccessors' (a nil value)

### ISSUE THREAD

**Issue by ericabxy** [BlackBulletIV/super-space-invader#7]
```
Error: lib/gfx/Spritemap.lua:17: attempt to call method 'enableAccessors' (a nil value)
stack traceback:
    lib/gfx/Spritemap.lua:17: in main chunk
    [C]: in function 'require'
    lib/gfx/init.lua:3: in main chunk
    [C]: in function 'require'
    main.lua:5: in main chunk
    [C]: in function 'require'
    [string "boot.lua"]:374: in function <[string "boot.lua"]:244>
    [C]: in function 'xpcall'

```

If it matters, this happens when using either Love version 0.9.2 or 0.10.1
Also I followed some of the instructions in Issue #5 , but none of them addressed this particular error

**Comment by akien-mga** [comment]
Yeah I've tried again to port this game to love 0.10.1 but hit the wall on this enableAccessors function. It seems to be specific to @BlackBulletIV's libs, and was maybe removed/broken at some time when porting ammo to love 0.10; fixing it is sadly above my non-existing love/lua skills :)

**Comment by ericabxy** [comment]
@akien-mga Is there a version of LOVE that super-space-invader still runs on? I'm willing to downgrade.

**Comment by mebens** [comment]
Hey guys, sorry about being inactive. I'm on a new rig and don't have my old data with me right now, however I do aim to update this. I remember fixing an issue like this before; it should be pretty simple, I've just gotta remember what I replaced enableAccessors with.

**Comment by akien-mga** [comment]
@EricxDu I believe the master branch should run properly on love 0.9.x, provided you don't update the submodules. At least it used to work for me :)

@BlackBulletIV Aweseome news :)
